tasks/broad/ImputationTasks.wdl

```diff
@@ -25,6 +25,85 @@ task CalculateChromosomeLength {
   }
 }
 
+task GetMissingContigList {
+  input {
+    File ref_dict
+    File included_contigs
+
+    String ubuntu_docker = "ubuntu:20.04"
+    Int memory_mb = 2000
+    Int cpu = 1
+    Int disk_size_gb = ceil(2*size(ref_dict, "GiB")) + 5
+  }
+
+  command <<<
+    grep "@SQ" ~{ref_dict} | sed 's/.*SN://' | sed 's/\t.*//' > contigs.txt
+    awk 'NR==FNR{arr[$0];next} !($0 in arr)' ~{included_contigs} contigs.txt > missing_contigs.txt
+  >>>
+
+  runtime {
+    docker: ubuntu_docker
+    disks: "local-disk ${disk_size_gb} HDD"
+    memory: "${memory_mb} MiB"
+    cpu: cpu
+  }
+
+  output {
+    Array[String] missing_contigs = read_lines("missing_contigs.txt")
+  }
+}
+
+task SortChunksByContigs {
+  input {
+    File vcfs_and_indices_and_contigs_list
+    File ref_dict
+  }
+
+  command <<<
+    set -e -o p
```

Comment:
```suggestion
    set -e -o -p
```

Reply:
not sure if you meant for this to be "-p" or if p was just a typo

Reply:
think this is supposed to be pipefail?

Reply:
haha yup :)